control-plane/control-plane-defaults/main.yaml

```diff
@@ -4,3 +4,11 @@ metal_control_plane_ingress_dns:
 metal_control_plane_stage_name: test
 metal_control_plane_namespace: metal-control-plane
 metal_control_plane_image_pull_policy: IfNotPresent
+
+# auditing
+metal_auditing_enabled: true
+metal_auditing_index_prefix: "auditing"
```

Comment:
The idea behind the `control-plane-defaults` is when a variable is getting used across many roles in the control-plane folder.

As this variable is only used in the `metal` role, you can move it to the defaults of this role.

Same applies for other variables, too.